js/controller.js

```diff
@@ -464,19 +464,19 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
     },
 
     onVolumeChanged: function (event, newVolume) {
-      if (typeof this.state.volumeState.volume === "number") {
-        this.state.volumeState.oldVolume = this.state.volumeState.volume;
-      }
       if (newVolume <= 0) {
```

Comment:
What happens if it's NaN or type of something else? What does this eval to?

Reply:
if newVolume is a type we aren't expecting, the state of the volume is going to be messed up since we are setting: `this.state.volumeState.volume = newVolume;` below. I believe this evaluates to false, though.

Reply:
coo beans